autorest/codegen/serializers/__init__.py

```diff
@@ -156,51 +157,97 @@ def _serialize_and_write_operations_folder(
                 operations_async_init_serializer.serialize(),
             )
 
-        for operation_group in code_model.operation_groups:
-            # write sync operation group and operation files
-            operation_group_serializer = OperationGroupSerializer(
+        if not code_model.options["combine_operation_files"]:
```

Comment:
@msyyc I don't think you need all this new code of adding operation groups serializers etc. I think all you need to do is configure the existing templates `operations_container_mixin.py.jinja2` and `operations_container.py.jinja2` to take in a list of operation groups. You will either pass in just one operation group, or all of them if you're combining. Then, in these templates, just do a for loop over the passed in operation groups and that should be it for the code

Reply:
It makes sense and I have commit the change.